tests/python/unittest/test_gluon_rnn.py

```diff
@@ -192,6 +196,59 @@ def test_rnn_cells():
     net.add(gluon.rnn.GRUCell(100, input_size=100))
     check_rnn_forward(net, mx.nd.ones((8, 3, 200)))
 
+    net = gluon.rnn.SequentialRNNCell()
+    net.add(gluon.rnn.ConvLSTMCell(100, input_shape=(200, 5, 7)))
+    net.add(gluon.rnn.ConvRNNCell(100, input_shape=(100, 5, 7)))
+    net.add(gluon.rnn.ConvGRUCell(100, input_shape=(100, 5, 7)))
+    check_rnn_forward(net, mx.nd.ones((8, 3, 200, 5, 7)))
+
+
+def test_convrnn():
+    cell = gluon.rnn.ConvRNNCell(hidden_size=10, input_shape=(3, 16, 10),
+                                 h2h_kernel=(3, 3), h2h_dilate=(1, 1),
+                                 i2h_kernel=(3, 3), i2h_stride=(1, 1),
+                                 i2h_pad=(1, 1), i2h_dilate=(1, 1),
+                                 prefix='rnn_')
```

Comment:
I find that the `conv_layout` has not been tested. It would be safer to test it.